uint8 BILLBOARD=0
uint8 BOUNDING_BOX=1
uint8 OBJECT=2
uint8 PLANE=3
uint8 PLANE_POLYGON=4
uint8 UNKNOWN_OBJECT=5
uint8 OBJECT_WITH_BOUNDING_BOX=6
